ui-core/src/main/java/com/adyen/checkout/ui/core/internal/ui/model/PhoneNumberInputData.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by oscars on 15/3/2023.
+ */
+
+package com.adyen.checkout.ui.core.internal.ui.model
+
+import androidx.annotation.RestrictTo
+import com.adyen.checkout.components.core.internal.ui.model.InputData
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+data class PhoneNumberInputData(
+    var countryCode: String = "",
+    var everythingAfterCountryCode: String = "",
```

Comment:
Nitpick: can we rename this? 😅 I don't have a much better suggestion, maybe something like `numberWithoutCountryCode`?

Reply:
Haha I still needed to change that indeed. There is not an official name for it since it consists out of multiple parts, but some sources just call it phone number. Should we call it `phoneNumber` as well?

Reply:
I think `phoneNumber` is clear. We can use it